### DISCUSSION TITLE
aux_resorb server not responding

### DISCUSSION THREAD

**Discussion by PriyomRoy** [alexisInSAR/EZ-InSAR#78] [Help]
The restitute orbit server https://s1qc.asf.alaska.edu/aux_resorb/ is non responsive for last few days. ISCE processing is failing.

**Comment by alexisInSAR** [comment]
Hi,

I had the same problem... The error is linked to the enormous size of the webpage due to the number of relative orbits. If I increase the time-out threshold in the EZ-InSAR scripts (test with Python), I get the same error. I suspect a timeout error on the server side... 

To be honest, I don't have a solution at the moment. I'm going to release a new version with an exception for the ASF server and the implementation of the Copernicus server. However, the server is more temperamental.

Kind regards, 

Alexis

PS: it sometimes works...
